client/lib/analytics/ad-tracking/google-analytics-4.ts

```diff
@@ -1,41 +1,49 @@
 import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
-import { GaPurchase } from '../utils/jetpack-cart-to-ga-purchase';
-import { GaItem } from '../utils/jetpack-product-to-ga-item';
+import { GaPurchase } from '../utils/cart-to-ga-purchase';
+import { GaItem } from '../utils/product-to-ga-item';
 import { TRACKING_IDS } from './constants';
 
 // Ensure setup has run.
 import './setup';
 
 export function setup( params: Gtag.ConfigParams ) {
-	// TODO: GA4 properties for WPCOM will be here
+	window.gtag( 'config', TRACKING_IDS.wpcomGoogleGA4Gtag, params );
 
 	if ( isJetpackCloud() ) {
 		window.gtag( 'config', TRACKING_IDS.jetpackGoogleGA4Gtag, params );
 	}
 }
 
-export function fireJetpackEcommercePurchase( purchase: GaPurchase ) {
+export function fireEcommercePurchase( purchase: GaPurchase, shouldSendToJetpack = false ) {
+	const send_to = shouldSendToJetpack
+		? TRACKING_IDS.jetpackGoogleGA4Gtag
+		: TRACKING_IDS.wpcomGoogleGA4Gtag;
 	window.gtag( 'event', 'purchase', {
-		send_to: TRACKING_IDS.jetpackGoogleGA4Gtag,
+		send_to,
 		...purchase,
 	} );
 }
 
-export function fireJetpackEcommerceAddToCart( item: GaItem ) {
+export function fireJetpackEcommerceAddToCart( item: GaItem, shouldSendToJetpack = false ) {
+	const send_to = shouldSendToJetpack
+		? TRACKING_IDS.jetpackGoogleGA4Gtag
+		: TRACKING_IDS.wpcomGoogleGA4Gtag;
 	window.gtag( 'event', 'add_to_cart', {
-		send_to: TRACKING_IDS.jetpackGoogleGA4Gtag,
+		send_to,
 		value: item.price,
 		currency: 'USD',
 		items: [ item ],
 	} );
 }
 
 export function firePageView( title: string, location: string, shouldSendToJetpack = false ) {
-	if ( shouldSendToJetpack ) {
-		window.gtag( 'event', 'page_view', {
-			send_to: TRACKING_IDS.jetpackGoogleGA4Gtag,
-			page_title: title,
-			page_location: location,
-		} );
-	}
+	const send_to = shouldSendToJetpack
```

Comment:
Just leaving here the remainder that we should make wpcom marketing team aware of the change in `page_view` events. Currently, wpcom property collets both cloud.jetpack.com and wordpress.com views.

Reference: https://github.com/Automattic/wp-calypso/blob/0e3346c8c6c3d0e59664110ff790da243e9b4bda/client/lib/analytics/ad-tracking/google-analytics.js#L86

Reply:
Thanks, this is good feedback. The Universal Analytics property is a bit "misleading" I think. I.e. we split out conversions in the code, but not page views. 

I also think the current UA code isn't 100% correct; it's not fully functional (I don't see hostnames containing jetpack in the UA view), all though we do send them. 

I think it's because of the cross-domain setup, that we ran earlier when we had discover.wordpress.com, blog.wordpress.com, etc. Those are all in sub-directories now (example: wordpress.com/discover), while earlier they had their own views.

I might be wrong though -- but @debbiengWP and I will verify the data once this is deployed.